packages/next/src/client/components/react-dev-overlay/ui/container/errors.tsx

```diff
@@ -89,18 +89,6 @@ export function Errors({
 }: ErrorsProps) {
   const dialogResizerRef = useRef<HTMLDivElement | null>(null)
 
-  useEffect(() => {
-    // Close the error overlay when pressing escape
-    function handleKeyDown(event: KeyboardEvent) {
-      if (event.key === 'Escape') {
-        onClose()
-      }
-    }
-
-    document.addEventListener('keydown', handleKeyDown)
-    return () => document.removeEventListener('keydown', handleKeyDown)
-  }, [onClose])
-
```

Comment:
This is already handled by the `Dialog` component. The logic here was also making the dialog dismiss when closing the Dev Tools popover with <kbd>Escape</kbd>